cmake/Modules/FindMKL.cmake

```diff
@@ -28,6 +21,7 @@ SET(MKL_LAPACK_LIBRARIES)
 SET(MKL_SCALAPACK_LIBRARIES)
 SET(MKL_SOLVER_LIBRARIES)
 SET(MKL_CDFT_LIBRARIES)
+SET(CAFFE2_USE_MKL OFF)
```

Comment:
Generally these modules match upstream versions and don't have any Caffe2/PyTorch specific code in them... is there a way to have this not set `CAFFE2_USE_MKL` directly?

Reply:
pls see blow comments.